Documentation/Storage-Configuration/Monitoring/ceph-dashboard.md

```diff
@@ -87,17 +87,18 @@ To prepare the Rook manager module to be used in the dashboard, modify your clus
 And apply the changes:
 
 ```console
-$ kubectl apply cluster.yaml
+$ kubectl apply -f cluster.yaml
 ```
 
 Once the Rook manager module is enabled as the orchestrator backend, there are two settings required for showing disk information:
-- `ROOK_ENABLE_DISCOVERY_DAEMON`: Set to `true` to provide the dashboard the information about physical disks. The default is `false`.
-- `ROOK_DISCOVER_DEVICES_INTERVAL`: The interval for changes to be refreshed in the set of physical disks in the cluster. The default is `60` minutes.  
+
```

Comment:
Maybe this extra line is not needed

Reply:
It is needed, see current v1.1.0 docs https://rook.github.io/docs/rook/v1.10/Storage-Configuration/Monitoring/ceph-dashboard/#visualization-of-physical-disks-section-in-the-dashboard :
![Rook docs formatting issue](https://user-images.githubusercontent.com/3718398/216995322-967fadf4-c409-42cb-b1da-cd7e50ef974c.png)